source/manual/grafana.html.md

```diff
@@ -0,0 +1,29 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Grafana
+section: Monitoring
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-08-16
+review_in: 6 months
+---
+
+Grafana lets us create nice dashboards using Graphite data.
+
+- [Production AWS Grafana](https://grafana.blue.production.govuk.digital)
+- [Staging AWS Grafana](https://grafana.blue.staging.govuk.digital)
+- [Integration AWS Grafana](https://grafana.blue.integration.govuk.digital)
+- [Production Carrenza Grafana](https://grafana.publishing.service.gov.uk)
+- [Staging Carrenza Grafana](https://grafana.staging.publishing.service.gov.uk)
+- [Integration Carrenza Grafana](https://grafana.integration.publishing.service.gov.uk)
```

Comment:
We don't have a Grafana instance in Carrenza integration anymore, this URL points to AWS.

Reply:
What was the Integration environment in Carrenza is now the CI environment, so this list is missing `https://ci-grafana.integration.publishing.service.gov.uk`.

Also, maybe having two levels of hierarchy would make this clearer, so:
```
 - Production
   - AWS
   - Carrenza
 - Staging
...
```